application/src/ext/java/org/opentripplanner/ext/siri/updater/azure/AbstractAzureSiriUpdater.java

```diff
@@ -34,6 +42,41 @@
 
 public abstract class AbstractAzureSiriUpdater implements GraphUpdater {
 
+  /**
+   *  custom functional interface that allows throwing checked exceptions, thereby
+   *  preserving the exception's intent and type.
+   */
+  @FunctionalInterface
+  interface CheckedRunnable {
+    void run() throws Exception;
+  }
+
+  private static final Set<ServiceBusFailureReason> RETRIABLE_REASONS;
+  static {
+    RETRIABLE_REASONS = Collections.unmodifiableSet(new HashSet<>(Arrays.asList(
```

Comment:
Is there a reason you're not using `Set.of`?

Reply:
Initially I had a compiler issue so this was a workaround, but you are right it should use Set.of. It is fixed now.